Orange/widgets/regression/owsgdregression.py

```diff
@@ -139,7 +144,8 @@ def set_data(self, data):
             self.apply()
 
     def apply(self):
-        loss = ["squared_loss", "huber", "epsilon_insensitive", "squared_epsilon_insensitive"][self.loss_function]
+        loss = ["squared_loss", "huber", "epsilon_insensitive",
+                "squared_epsilon_insensitive"][self.loss_function]
         penalty = ["l1", "l2", "elasticnet"][self.penalty_type]
         learning_rate = ["invscaling", "constant"][self.learning_rate]
```

Comment:
But in sklearn library the parameters have to be like that. e.g `l1` and `l2` are predefined in sklearn library.
I can't change these in the **apply()** function as these are inbuilt.
What do you suggest?